src/Server.cpp

```diff
@@ -63,6 +92,21 @@ void Server::init( void ) {
     std::cout << BOLDGREEN << "Successfuly created listening socket!" << RESET << std::endl;
 }
 
+/**
+ * @brief Once the Listening Socket and the commands are initialized use this function 
+ * @brief to handle new connections and send/recieve msgs. This function needs a vector of FDS
+ * @brief to keep track of the active clients and closed clients (_pollFds)
+ * 
+ * @note This functio will monitor 2 events:
+ * @note [POLLIN] - data can be read/recieved from the client without blocking
+ * @note     - once this event is detected, the server attempts to read data using _handleClientRecv
+ * @note     - if recv returns 0 -> client closed their side of the connection -> purge client
+ * @note     - if recv > 0 -> parse the data -> add msg to the queue of the client
+ * @note  
+ * @note [POLLOUT] - data can be sent to the client without blocking
+ * @note     - once this event is detected if there are messages in the queue -> attempt to send them
+ * 
+ */
 void Server::run( void ) {
```

Comment:
if else conditions is gonna look cleaner then you can remove repetitive lines of code.